src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -1642,9 +1618,7 @@ void EmitWhenHasCapture()
                         writer.WriteLine($"matchIndex = base.MatchIndex({capnum});");
                         using (EmitBlock(writer, $"for (int i = 0; i < matchLength; i++)"))
                         {
-                            using (EmitBlock(writer, caseInsensitive ?
-                                $"if (_textInfo.ToLower(inputSpan[matchIndex + i]) != _textInfo.ToLower(inputSpan[pos - matchLength + i]))" :
-                                $"if (inputSpan[matchIndex + i] != inputSpan[pos - matchLength + i])"))
```

Comment:
Shouldn't we also be deleting all of this?
https://github.com/dotnet/runtime/blob/d7b0629a0ccd873b6ac7e1b39b5c5b062eae3155/src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs#L182-L202

Reply:
Oops yes I forgot, thanks for pointing it out.